src/bindings/python/src/openvino/frontend/tensorflow/__init__.py

```diff
@@ -14,6 +14,7 @@
 
 try:
     from openvino.frontend.tensorflow.py_tensorflow_frontend import ConversionExtensionTensorflow as ConversionExtension
+    from openvino.frontend.tensorflow.py_tensorflow_frontend import OpExtensionTensorflow as OpExtension
```

Comment:
Is it possible that we can have ConversionExtension but don't have OpExtension?

Reply:
OpExtension is derived from ConversionExtension, so they both should exist